ci/build_deps_img.sh

```diff
@@ -3,7 +3,11 @@
 # script to setup new image (best to do on each intellij version bump)
 
 USERNAME=odisseus
-IJ_VERSION=202.6948.69
+IJ_VERSION=202.7319.50
```

Comment:
if we remove intellij from image, we don't need to put any version anywhere regarding the image, right?

Reply:
IntelliJ is still used to build the image, so a change of its version might affect the contents of the image. Therefore, I think it is better to keep tagging these images with the version of IntelliJ that was used to build them.

I have also added the "latest" tag for convenience, even though it is considered [harmful](https://vsupalov.com/docker-latest-tag/).

Reply:
But the whole idea is to have the image intellij-independent. We delete everything intellij version related, we mostly set everything up to get the sbt and pants build setup up to speed.

Reply:
ok